### ISSUE TITLE
PNG images are not loaded, as they are corrupted

### ISSUE THREAD

**Issue by instruccionesaldorso** [joppnet/jn_lighterbox#19]
After upgrading the extension from 1.2.5 to 1.3.1, I can see the PNG images contained there are corrupted and are not loaded:
https://github.com/joppnet/jn_lighterbox/tree/master/Resources/Public/Images/

That causes the lightbox cannot load the close button, etc.

Would you mind fixing this in latest version and launch a new 1.3.2 version?

**Comment by linawolf** [comment]
In my projects the frontend looks fine, cannot reproduce

**Comment by instruccionesaldorso** [comment]
@oliverschloebe @linawolf Well, lightbox.min.css has e.g. this rule:

```
.lb-nav a.lb-prev{
  ...
  background:url(../Icons/prev.png) left 48% no-repeat;
  ...
}
```

but this icon is corrupted, as you can see here: https://github.com/joppnet/jn_lighterbox/blob/master/Resources/Public/Images/prev.png

When I open the lightbox and there are several images attached to the content element, the usual arrow-right-icon cannot be seen.

Have you tried in an incognito window, with the cache flushed?

**Comment by oliverschloebe** [comment]
> When I open the lightbox and there are several images attached to the content element, the usual arrow-right-icon cannot be seen.

This is weird as the arrows show up for me (when hovering left or right over the image)
![image](https://github.com/user-attachments/assets/3335744a-18fe-4e09-9b6e-72b34aa219ff)

Also when opening the image in a new tab it renders fine for me:
![image](https://github.com/user-attachments/assets/bb32d3c1-8463-4b9e-8e5d-43bbe80684fb)


What OS are you running TYPO3 on?

**Comment by instruccionesaldorso** [comment]
@oliverschloebe I'm impressed that you can see the images.
On your network panel in your browser, the prev.png and next.png files are previewed properly?
Have you just deleted the whole folder `public/_assets` and recreated it again, just in case the symlink from the extension pointed to an old folder under e.g. `public/typo3conf/ext`?

My OS is: Mac OS X - Sequoia 15.1.1
I'm using latest DDEV version: 1.24.1
I've upgraded also to latest EXT:jn_lighterbox 1.3.5 version.

**Comment by oliverschloebe** [comment]
> @oliverschloebe I'm impressed that you can see the images. On your network panel in your browser, the prev.png and next.png files are previewed properly? Have you just deleted the whole folder `public/_assets` and recreated it again, just in case the symlink from the extension pointed to an old folder under e.g. `public/typo3conf/ext`?

Yep they're previewed properly in the network panel, even after deleting temporary assets via Maintenenace Module:

![image](https://github.com/user-attachments/assets/d02ddf54-b8c2-4f0e-b354-eaf13235d405)

**Comment by w4jzscheile** [comment]
@instruccionesaldorso have the same issue, also for me only loading.gif is working the three others seems all corrupt. Also the size of the images is different to the screenshot of @oliverschloebe .

![Image](https://github.com/user-attachments/assets/5af80de3-6d29-45a0-b5dc-aba4bb323fa9)

Images can't also not open by any graphic program or tool.

```
% identify Resources/Public/Images/next.png 
identify: improper image header `Resources/Public/Images/next.png' @ error/png.c/ReadPNGImage/3942.
% identify Resources/Public/Images/prev.png 
identify: improper image header `Resources/Public/Images/prev.png' @ error/png.c/ReadPNGImage/3942.
% identify Resources/Public/Images/close.png 
identify: improper image header `Resources/Public/Images/close.png' @ error/png.c/ReadPNGImage/3942.
% identify Resources/Public/Images/loading.gif 
Resources/Public/Images/loading.gif[0] GIF 32x32 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.001
Resources/Public/Images/loading.gif[1] GIF 25x32 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[2] GIF 25x32 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[3] GIF 28x32 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[4] GIF 31x32 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[5] GIF 32x31 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[6] GIF 32x28 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[7] GIF 32x26 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[8] GIF 32x25 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[9] GIF 32x28 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[10] GIF 32x31 32x32+0+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[11] GIF 31x32 32x32+1+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[12] GIF 28x32 32x32+4+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[13] GIF 26x32 32x32+6+0 8-bit sRGB 64c 0.000u 0:00.002
Resources/Public/Images/loading.gif[14] GIF 25x32 32x32+7+0 8-bit sRGB 64c 8475B 0.000u 0:00.002
identify: corrupt image `Resources/Public/Images/loading.gif' @ error/gif.c/PingGIFImage/949.
```

**Comment by stm-cmd** [comment]
Confirm issue. Same Problem on our site.

**Comment by oliverschloebe** [comment]
Just released 1.3.6, please check if the images are fixed for you now.
